packages/inngest/src/components/InngestFunction.ts

```diff
@@ -363,6 +363,40 @@ export namespace InngestFunction {
       period: TimeStr;
     };
 
+    /**
+     * Rate limit workflows, only running them a given number of times (limit) per
```

Comment:
Does "workflow" mean "function run"? I don't think we use the term "workflow" much anymore

```suggestion
     * Rate limit function runs, only running them a given number of times (limit) per
```